### PR TITLE
Fix NPE when rewards section not found in file, #449

### PR DESCRIPTION
**Pull request by montlikadani**
This fixes [#449](https://plajer.xyz/errorservice/viewer.php?id=449#)

_Version up randomly._

### CODE REVIEW

File: src/main/java/pl/plajer/villagedefense/handlers/reward/RewardsFactory.java [hunk 1]
```diff
@@ -113,6 +113,10 @@ private void registerRewards() {
     if (!enabled) {
       return;
     }
+    if (!config.contains("rewards")) {
+      Debugger.debug(LogLevel.WARN, "[RewardsFactory] Rewards section not found in the file. Rewards not loaded.");
```
Comment (Plajer) [inline src/main/java/pl/plajer/villagedefense/handlers/reward/RewardsFactory.java]: Rewards won't be loaded

File: pom.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
     <groupId>pl.plajer</groupId>
     <artifactId>villagedefense</artifactId>
-    <version>4.0.2</version>
+    <version>4.0.1</version>
```
Comment (Plajer) [inline pom.xml]: Why did you downgrade version?

**Comment by Plajer** [comment]
Good pull but maybe I suggest you to use `development` branch instead to have latest code changes?

File: pom.xml (hunk 2 above)
Comment (montlikadani) [reply pom.xml]: I didn't touch it, I just went to the development branch.

**Pull request closed without merging**